pkg/config/config.go

```diff
@@ -0,0 +1,373 @@
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/BurntSushi/toml"
+	"github.com/containers/image/pkg/sysregistriesv2"
+	"github.com/containers/image/types"
+	units "github.com/docker/go-units"
+	selinux "github.com/opencontainers/selinux/go-selinux"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	cgroupManager          = "systemd"
+	DefaultApparmorProfile = "container-default"
+	// DefaultPidsLimit is the default value for maximum number of processes
+	// allowed inside a container
+	DefaultPidsLimit = 1024
+	// DefaultLogSizeMax is the default value for the maximum log size
+	// allowed for a container. Negative values mean that no limit is imposed.
+	DefaultLogSizeMax = -1
+	OCIBufSize        = 8192
```

Comment:
I think we should import this from 
`containers/conmon/runner/config` as it's a default that also configures conmon

Reply:
We need this the other way around.  conmon should import this from here.